fe-nc-news/src/articlePage.css

```diff
@@ -10,4 +10,18 @@
     display: inline-block;
   font-size: 20px;
   padding: 10px;
+}
+
+.authorVotes{
+  background-color: rgb(40, 47, 56); 
+  list-style-type: none;
+  text-align: center;
+  margin: 0;
+  padding: 0;
+}
+
+.authorVotes p {
+  display: inline-block;
+  font-size: 15px;
+  padding: 5px;
```

Comment:
In the comments, I can see the author and votes info in dark mode, but in light mode it's black on a dark colour and impossible to read. Have a look at the base styling or your own styles and make sure it's readable in both modes